### PR TITLE
[MB-4452] Remove Loki

### PR DESCRIPTION
**Pull request by suzubara**
## Description

This PR completes the last AC of https://dp3.atlassian.net/browse/MB-4452 and deletes any code or files related to Loki & running Storybook in Docker (which is no longer needed).

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
echo "Code goes here"
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](tbd) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -11,208 +11,204 @@ If you want to see if scripts are not listed in this file you can run
...
+| ------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| `run-e2e-test`                        | Runs cypress tests with interactive GUI                                                          |
+| `run-e2e-test-docker`                 | Runs cypress tests entirely inside docker containers like in CircleCI                            |
+| `run-e2e-mtls-test-docker`            | Runs integration tests for mtls endpoints inside docker containers like in CircleCI              |
+| `run-server-test`                     | Run golang server tests                                                                          |
 | `run-server-test-in-circle-container` | Executed in docker-compose.circle.yml to run the `make server_test` task in a CircleCI container |
-| `run-storybook-tests` | Runs the storybook tests in docker containers for CircleCI testing. |
-| `start-storybook-tests` | Intended to run inside a docker container, will wait for a 'storybook' container to have started before running the tests. |
```
Comment (gidjin) [inline scripts/README.md:131]: Glad you removed these, I wonder why the whole file changed, I assume it was related to your editor. Though I like it better.

**Review by gidjin — APPROVED** [review]
I've looked a few times through this, and I don't see any loki or storybook docker stuff that was missed. You got everything I could think of. 🚀

File: scripts/README.md (hunk 1 above)
Comment (suzubara) [reply scripts/README.md:131]: Yeah I have auto formatting turned on for markdown files... I also like it better :D

**Review by amanda5iu — APPROVED** [review]
Get outta here Loki

**Pull request merged**